public/static/docs/command-reference/metrics/diff.md

```diff
@@ -0,0 +1,109 @@
+# metrics diff
+
+Find and print changes in [metrics](/doc/command-reference/metrics#description)
+between <abbr>project</abbr> versions.
+
+## Synopsis
+
+```usage
+usage: dvc metrics diff [-h] [-q | -v]
+                        [--targets [TARGETS [TARGETS ...]]]
+                        [-t TYPE] [-x XPATH] [-R] [--show-json]
+                        [a_ref] [b_ref]
+
+positional arguments:
+  a_ref     Git reference from which diff is calculated. If
+            omitted, `HEAD` (latest commit) is used.
+  b_ref     Git reference to which diff is calculated. If omitted,
+            the current workspace is used instead.
+```
+
+## Description
+
+Calculates the numeric difference (delta) between a metric's value in two
+different
+[Git references](https://git-scm.com/book/en/v2/Git-Internals-Git-References)
+(such as a branch name, a tag, or a commit hash) for all metrics in the
+<abbr>project</abbr> by examining all of its
+[DVC-files](/doc/user-guide/dvc-file-format).
+
+Note that `a_ref` and `b_ref` have different defaults than those in `dvc diff`,
```

Comment:
let's keep it for now, but move all the way down. First we need to focus on the command, not on the differences with other commands

Reply:
Maybe I should remove it actually, per https://github.com/iterative/dvc.org/pull/933#discussion_r371379281.

Reply:
Done.